docs/iam-usage.rst

```diff
@@ -0,0 +1,124 @@
+Using IAM For Users
+===================
+
+.. note:: This document does not explain how Google Cloud IAM works,
+   for that, please check out `the docs <https://cloud.google.com/iam/docs/>`_.
+
+Resources that implement the IAM interface provide the following methods:
+
+Policy Editing Methods
+----------------------
+
+>>> resource.add_roles(iam.user('alice@example.com'), iam.roles.OWNER.name, iam.roles.EDITOR.name)
```

Comment:
Is it actually necessary to say "this is a user" or "this is a group", could just passing a string suffice?